jmh/src/jmh/java/org/roaringbitmap/map/MapBenchmark.java

```diff
@@ -27,12 +27,7 @@ public static int inttointmap(int x) {
   @Benchmark
   public RoaringBitmap testMap(BenchmarkState benchmarkState) {
     final RoaringBitmap answer = new RoaringBitmap();
-    benchmarkState.bitmap.forEach(new IntConsumer() {
-      @Override
-      public void accept(int value) {
-        answer.add(inttointmap(value));
-      }
-    });
+    benchmarkState.bitmap.forEach((IntConsumer) value -> answer.add(inttointmap(value)));
```

Comment:
I don't get the need for `(IntConsumer)`

Reply:
See https://github.com/RoaringBitmap/RoaringBitmap/pull/455/files#r571490784